INFO:root:Program started
INFO:root:Robot set up!
INFO:root:Odometry was started, PID: 5110:
INFO:root:Initial location: 0.2, 1.8, 0.0
WARNING:root:Traceback (most recent call last):
  File "/home/pi/oscar/main.py", line 53, in main
    oscar.navigateMap([0, 4], [4, 1])
  File "/home/pi/oscar/Robot.py", line 94, in navigateMap
    arr_pos = sage.pos2array(size, [self.x.value, self.y.value]) # calcula la pos que tiene en el mapa
TypeError: pos2array() missing 1 required positional argument: 'pos'

WARNING:root:Abruptly stopped the robot!
INFO:root:Odometry was stopped
